rewrite-java/src/main/java/org/openrewrite/java/internal/template/JavaTemplateParser.java

```diff
@@ -91,6 +92,18 @@ public JavaTemplateParser(Supplier<JavaParser> parser, Consumer<String> onAfterV
         }).get(0);
     }
 
+    public J parseExpression(String template) {
```

Comment:
Expressions can refer to variables/types/methods in scope. The complexity of expression replacement is similar to that of statement replacement.